src/Common/src/Interop/Unix/libc/Interop.sockaddr.cs

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Runtime.InteropServices;
+using System.Text;
+
+using in_port_t = System.UInt16;
+using sa_family_t = System.UInt16;
+
+internal static partial class Interop
+{
+    internal static partial class libc
+    {
+        public const sa_family_t AF_INET = 2;   // IP protocol family.
+        public const sa_family_t AF_INET6 = 10; // IP version 6.
```

Comment:
When do we plan to adopt Nick's interop models which deals with these sorts of issues?